### PR TITLE
Add presentation to resident-card-tutorial

### PR DESCRIPTION
**Pull request by jrhender**
This PR adds presentation of the resident card credential and fixes the issuance to work given the current app.

### CODE REVIEW

**Review by Harasz — APPROVED** [review]

File: apps/vc-api/docs/tutorials/resident-card-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+            "https://www.w3.org/2018/credentials/examples/v1"
+        ],
+        "type":[
+            "VerifiablePresentation"
+        ],
+        "verifiableCredential":[
+            "<FILL WITH VC RECEIVED FROM AUTHORITY>"
+        ],
+        "holder": "<FILL WITH RESIDENT DID>"
+    },
```
Comment (JGiter) [inline apps/vc-api/docs/tutorials/resident-card-tutorial.md:372]: Is this other request format than described in https://w3c-ccg.github.io/vc-api/#continue-exchange? It `proof` property is moved from presentation to options object

Comment (jrhender) [reply apps/vc-api/docs/tutorials/resident-card-tutorial.md:372]: Yes, this request isn't for continuing the exchange, but rather proving the presentation: https://w3c-ccg.github.io/vc-api/#prove-presentation

File: apps/vc-api/docs/tutorials/resident-card-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+From a technical point of view, in this tutorial, we have access to the server API but no mobile wallet is available. So we will use the server API for both roles of the portal and the resident's mobile wallet.
+
+We will follow the following steps in this tutorial, covering both credential issuance and presentation:
+- [Resident action] Create a DID for the resident using the "did:key" DID method
+- [Authority portal action] Create a DID for the authority portal using the "did:key" DID method
+- [Authority portal action] Issue a "PermanentResidentCard" credential to the resident's DID
+- [Authority portal action] Configure a credential exchange which requests the "PermanentResidentCard" credential 
+- [Authority portal action] Transmit an exchange invitation to the resident as a QR code or a deep link
+- [Resident action] Initiate the exchange and obtain a [Verifiable Presentation Request](https://w3c-ccg.github.io/vp-request-spec/) in return
+- [Resident action] Create a Verifiable Presentation which contains the "PermanentResidentCard" credential
```
[the remark this answers is not in the document]
Comment (jrhender) [reply apps/vc-api/docs/tutorials/resident-card-tutorial.md:73]: Hmm, I'm not really following. You mean how does the resident know which credentials satisfy the VP Request?

File: apps/vc-api/docs/tutorials/resident-card-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,520 @@
...
+
+We therefore assume that the resident is connected to the autority portal and that the information contained in the context about the resident is available in the autority portal database.
+
+### Business workflows
+
+#### Issuance Business workflow
+1. The authority portal authenticates the current user as a known resident.
+1. The resident requests a "PermanentResidentCard" credential on the autority portal and provides a DID of their choosing.
+1. The authority portal issues a "PermanentResidentCard" containing the resident's information to that DID and displays a QR code allowing the resident's wallet to receive the credential.
+1. The resident's scans the QR code with their mobile wallet. The wallet then contacts the autority portal again to receive the "PermanentResidentCard" credential.
```
[the remark this answers is not in the document]
Comment (jrhender) [reply apps/vc-api/docs/tutorials/resident-card-tutorial.md:56]: Thanks, I updated the PR

[the remark this answers is not in the document]
Comment (jrhender) [reply apps/vc-api/docs/tutorials/resident-card-tutorial.md]: Although I kept `their` as it is gender neutral (as we don't know if the resident is male or female at this point)

File: apps/vc-api/docs/tutorials/resident-card-tutorial.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JGiter) [reply apps/vc-api/docs/tutorials/resident-card-tutorial.md:73]: Nevermind. I read this step as if resident sign Verifiable Presentation but in fact he ask to sign

**Review by JGiter — APPROVED** [review]

**Pull request merged**